vars/runPCT.groovy

```diff
@@ -127,7 +127,7 @@ def call(Map params = [:]) {
                         }
                         def command = "JENKINS_WAR_PATH=${warAbsolutePath} run-pct ${pctBranchOptions.join(' ')}"
                         if (!javaOptions.isEmpty()) {
-                            command = "JAVA_OPTS=${javaOptions.join(' ')} ${command}"
+                            command = "JAVA_OPTS=\"${javaOptions.join(' ')}\" ${command}"
```

Comment:
NIT: generally I find that using the nice triple single/double quotes makes code more readable avoiding to use `\`.

Here this would be:

```groovy
command = '''JAVA_OPTS=${javaOptions.join(' ')} ${command} '''
```

Reply:
Oh wait, we need the expression to be interpreted, so we want triple double `"""` instead

```groovy
command = """JAVA_OPTS="${javaOptions.join(' ')}" ${command} """
```

Reply:
Yeah, I am with @batmat here, it makes it much more readable than escaping characters